src/analyze/analyze-capability.c

```diff
@@ -48,5 +48,9 @@ int verb_capabilities(int argc, char *argv[], void *userdata) {
 
         pager_open(arg_pager_flags);
 
-        return table_print(table, NULL);
+        r = table_print(table, NULL);
+        if (r < 0)
+                return r;
+
+        return 0;
```

Comment:
ditto (and everywhere else).

i.e. if this is propagated as is now if > 0 we should probably use only EXIT_xyz names for returning here